### ISSUE TITLE
Rename "Environment Settings" patch to "Fog"

### ISSUE THREAD

**Issue by mattatgit** [vizorvr/vizor#825]
![screenshot 2015-11-19 12 26 46](https://cloud.githubusercontent.com/assets/6858525/11259792/d7b24110-8eb8-11e5-841c-c099620d7cb4.png)

This should apply to nested patches within as well

**Comment by mattatgit** [comment]
Is there any reason to not do this ? @kschzt @kkananen

**Comment by kschzt** [comment]
are there any other `environment` details we'd like to add to this? @kkananen

**Comment by mattatgit** [comment]
After some more thought, "Distance Fog" would make it most understandable to new users. Can we do this ?

**Comment by mattatgit** [comment]
Actually what i would like to do before we push the new blog posts live is update the default scene to this: 
![screenshot 2016-02-02 17 46 02](https://cloud.githubusercontent.com/assets/6858525/12741921/f368a8ca-c9d4-11e5-85c0-3be40017cf76.png)

1. i want to rename fog to what it should be to make sense (environment settings = NG)
2. i want to order the patches in the scene patch in a way that makes sense to new users (ie fog, sky, ground, light, object)
3. i want to use the hemisphere light as it is currently the nicest looking light we have in Vizor, with the most control.
4. I want to name the patches as shown in the the screenshot (ie Render Camera > Renderer, because that patch _contains_ the VR camera but is essentially the output to the default Vizor renderer.
You guys ok with this ? @fthr @kschzt @kkananen 

I would also like to include the new light icons, so they are unified with the VR cam icon. I will ask Jeremy now for another one for the Hemisphere light.

I guess we can do some of this in our next pass on the patch list @fthr

**Comment by fthr** [comment]
What about using the planet earth one, one you had made earlier?

**Comment by mattatgit** [comment]
Yeah we could put that in. Only problem is just that we don't have shadows yet, and it looks really odd floating there without a shadow. I had a fake shadow made with a flat sphere in there before, but i don't think we want to be putting fake shadows in our default scenes.
